Update tutorials to latest monthly tag (#607)
substrate-developer-hub/substrate-docs: v3/tutorials/01-create-your-first-substrate-chain/index.mdx

1. Clone the node template repository using the version `latest` branch by running the following command:

   ```

>    git clone https://github.com/substrate-developer-hub/substrate-node-template

Reviewer: this is not as we discussed right? we in fact want yo use the `latest` tag explicitly. that we we can iterate on `main` and when ready, shift all contact by just moving the `latest` tag.

Reviewer: we also absolutely should use depth = 1. why not?

Author: As per my note above:
![Screen Shot 2021-11-26 at 2 28 03 PM](https://user-images.githubusercontent.com/23283108/143588186-9580e43e-9cf9-4bc6-be2e-dc64c6266551.png)

Yes I agree, using `--depth 1` is not a bad idea.

Reviewer: This line points to the default branch (`master`) and is incorrect. It should be `latest` in this PR ***OR*** `monthly-2021-11-1`

Reviewer: Are you waiting on some template companion to get the tag in place first? If so, please mention it explicitly in the original PR comment :pray:

Author: Right now master is using `monthly-2021-11-1` and `latest` is using `monthly-2021-10`. So given this PR is work done in parallel to updating our solutions to `monthly-2021-11-1`, I wanted to make sure the write-up corresponds to the solution code. 

You've mentioned different plans for ensuring our template is always using `latest` which is what we need to follow up with. But for now, at least with this PR, our users won't have issues when going through tutorials.

As for figuring out the way to maintain our template to `latest`, we need to prioritize this as top priority this week so that we can drop the need to keep tagging solutions and write-ups each month.

1. Clone the Front-end Template repository by running the following command:

   ```

>    git clone https://github.com/substrate-developer-hub/substrate-front-end-template

Author: ```suggestion
   git clone -b latest --depth 1 https://github.com/substrate-developer-hub/substrate-front-end-template
```

1. Start the node in development mode by running the following command:

   ```
   ./target/release/node-template --dev
   ```

   The `node-template` command-line options specify how you want the running node to operate.

>    In this case, the `--dev` option specifies that the node is run as a developer node chain specification.

Reviewer 2: ```suggestion
   In this case, the `--dev` option specifies that the node runs in developer mode using the predefined `development` chain specification.
```

>    By default, this also implies that the node will delete all active data—such as keys, blockchain database, and networking information when you stop the node by pressing Control-c.

Reviewer 2: ```suggestion
   By default, this option also deletes all active data—such as keys, the blockchain database, and networking information when you stop the node by pressing Control-c.
```

>    This ensures you have a clean working state any time you stop and restart the node.

Reviewer 2: ```suggestion
   Using the `--dev` option ensures that you have a clean working state any time you stop and restart the node.
```

After a successful transfer, you can continue to explore the front-end template components or
stop the local Substrate node the state changes you made.

> With `--dev` flag specified when running the Node Template, stopping the local node stops the

Reviewer 2: ```suggestion
Because you specified the `--dev` option when you started the node, stopping the local node stops the blockchain and purges all persistent block data so that you can start with a clean state next time you start the node.
```

> blockchain and purge all persistent block data that goes with it so you can start with a clean new

Reviewer 2: ```suggestion
```

> state next time you start again.

Reviewer 2: ```suggestion
```

After a successful transfer, you can continue to explore the front-end template components or
stop the local Substrate node the state changes you made.

> Because you specified the `--dev` option when you started the node, stopping the local node stops the blockchain and purges all persistent block data so that you can start with a clean state next time you start the node.

Reviewer: https://github.com/paritytech/substrate/pull/9938
We want to maybe surface this? As some users may depend on this behavior. 

This should not live in tutorials... but where? Some FAQ or Troubleshooting? 
Or rely on upstream release notes?
